python/pyspark/sql/connect/proto/catalog_pb2.py

```diff
@@ -16,13 +16,19 @@
 #
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: spark/connect/catalog.proto
+# Protobuf Python Version: 5.28.3
 """Generated protocol buffer code."""
-from google.protobuf.internal import builder as _builder
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import runtime_version as _runtime_version
 from google.protobuf import symbol_database as _symbol_database
+from google.protobuf.internal import builder as _builder
 
+_runtime_version.ValidateProtobufRuntimeVersion(
+    _runtime_version.Domain.PUBLIC, 5, 28, 3, "", "spark/connect/catalog.proto"
```

Comment:
Actually let's don't do this for now. Seems like we're losing the compatibility in `protobuf`, and for some reasons this specific version make the connect tests failed in Mac .. 

I am taking a look now because there look other related PRs.

I will revert this first and investigate further 🙏

Reply:
Alright, sorry for forth and back. It was because of https://github.com/apache/spark/commit/6c4d3446b4457992eca03dea555a7d450d9b42af .. Let me get this one back since we explicitly specified it in `dev/requirements.txt`, it should be fine.

Reply:
Thanks for bearing with me 🙏

Reply:
Thank you for your investigation :) @HyukjinKwon